pr-fetch/lib/main.js

```diff
@@ -35,7 +35,7 @@ function run() {
             const headBranch = pr.head.ref;
             const headCloneURL = pr.head.repo.clone_url;
```

Comment:
Maybe rather than creating a new variable, just use the existing one?

```suggestion
            const headCloneURL = pr.head.repo.clone_url.replace("https://", `https://x-access-token:${token}@`);
```

Reply:
I meant to suggest this change in the typescript code, but the same idea applies there.

Reply:
👍  But then we should also update the pr-push action for consistency - is that something you'd like me to do?

Reply:
Sure, sounds good!

Reply:
I made the changes in the online editor, so it needed 4 additional commits. But I saw that you squashed the commits in the other PR, so I hope it's not a problem. 👍 

I've validated that this works as expected with private repos.